We sort of sabotage the test here, changing the result when the local
portmapper is running but won't allow us to register so that it's treated the
same as a portmapper-not-running case.

--- krb5/src/lib/rpc/unit-test/server.c
+++ krb5/src/lib/rpc/unit-test/server.c
@@ -116,6 +116,7 @@ main(int argc, char **argv)
      if (!svc_register(transp, RPC_TEST_PROG, RPC_TEST_VERS_1,
 		       rpc_test_prog_1_svc, prot)) {
 	  fprintf(stderr,
+		  "Cannot register service: " /* don't bail fatally just because rpcbind isn't obliging us */
 		  "unable to register (RPC_TEST_PROG, RPC_TEST_VERS_1, %s).",
 		  prot == IPPROTO_TCP ? "tcp" : "udp");
 	  exit(1);
